### PR TITLE
v 0.1

### PR DESCRIPTION
**Pull request by tomaszrondio**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
+node_modules/
```
Comment (ahilles107) [inline .gitignore]: Only node_modules i sass-cache should stay here, rest should go to your global .gitingnore file.

File: amp/amp-theme/css/amp-boilerplate-noscript.html.twig [hunk 2]
```diff
@@ -0,0 +1 @@
+<noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
```
Comment (ahilles107) [inline amp/amp-theme/css/amp-boilerplate-noscript.html.twig]: missing new line at end of file (any many more other files)

File: views/base.html.twig [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+</head>
+<body>
+  <header class="header">
+    <button class="header__hamburger"></button>
+    <div class="header__logo">
+      <a href="/" class="logo"></a>
+    </div>
+
+
+    {# {% container 'mainNav' with {
```
Comment (ahilles107) [inline views/base.html.twig]: why it's outside of container?

File: views/base.html.twig [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  <div class="overlay"></div>
+  {{ knp_menu_render('mainNavigation',{'template':'widgets/mobileNav.html.twig'}) }}
+
+
+{% block body %}{% endblock %}
+
+
+  <script type="text/javascript">
+    WebFontConfig = {
+      google: { families: [ 'Open+Sans:300,300i,400,400i,600,600i,700,800' ] }
```
Comment (ahilles107) [inline views/base.html.twig]: We need to use Open Sans? We can't use system fonts? It would improve load time a lot.

File: views/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+
+
+<section class="headerMargin">
+
+	{% cache 'front' { time:600 } %}
+
+	{% gimmelist article from articles|start(0)|limit(11) %}
```
Comment (ahilles107) [inline views/index.html.twig]: Code indentation and formating looks very bad here. Please fix it.

File: views/partials/article-gallery.html.twig [hunk 6]
```diff
@@ -0,0 +1,19 @@
+
+<script>
+ var galleryLinks = [];
+
+  {% gimmelist media from articleMedia with {'article': article} %}
+    {% if media.renditions is iterable %}
```
Comment (ahilles107) [inline views/partials/article-gallery.html.twig:5]: there is new way for fetching rendition by name: http://superdesk-web-publisher.readthedocs.io/en/latest/templates_system/article_media.html

File: views/partials/commentBox.html.twig [hunk 7]
```diff
@@ -0,0 +1,15 @@
+{% container 'commentBox' with {
+  'cssClass': 'box centerBoth'
+  }
```
Comment (ahilles107) [inline views/partials/commentBox.html.twig]: if you start block in one line - close it also like that.

File: theme.json [hunk 8]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "swp/default-theme",
```
Comment (takeit) [inline theme.json]: the name should be `swp/magazine-theme`

**Review by takeit — CHANGES_REQUESTED** [review]

File: views/base.html.twig (hunk 4 above)
Comment (tomaszrondio) [reply views/base.html.twig]: ask Backo if we need open sans. Removing it won't improve load time that much. It is loaded asynchronously and cached.

File: views/partials/article-gallery.html.twig (hunk 6 above)
Comment (tomaszrondio) [reply views/partials/article-gallery.html.twig:5]: that is version 0.1

**Review by takeit — APPROVED** [review]

**Pull request merged**